source/conf.py

```diff
@@ -124,6 +123,7 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
+html_css_files = ["css/custom.css"]
```

Comment:
What version of Sphinx have you tried this with? I think in the past this setting didn't exist. I might be wrong on that, though.

Reply:
According to the [documentation](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_css_files), it has been added in version 1.8.
I don't know which exact version I used though right now, I'll assume the newest one as I just followed the requirements.txt file.